airbyte-integrations/connectors/source-gcs/integration_tests/spec.json

```diff
@@ -0,0 +1,255 @@
+{
+  "documentationUrl": "https://docs.airbyte.com/integrations/sources/gcs",
+  "connectionSpecification": {
+    "title": "Config",
+    "description": "NOTE: When this Spec is changed, legacy_config_transformer.py must also be\nmodified to uptake the changes because it is responsible for converting\nlegacy GCS configs into file based configs using the File-Based CDK.",
+    "type": "object",
+    "properties": {
+      "start_date": {
+        "title": "Start Date",
+        "description": "UTC date and time in the format 2017-01-25T00:00:00.000000Z. Any file modified before this date will not be replicated.",
+        "examples": ["2021-01-01T00:00:00.000000Z"],
+        "format": "date-time",
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}.[0-9]{6}Z$",
+        "pattern_descriptor": "YYYY-MM-DDTHH:mm:ss.SSSSSSZ",
+        "order": 1,
+        "type": "string"
+      },
+      "streams": {
+        "title": "The list of streams to sync",
+        "description": "Each instance of this configuration defines a <a href=https://docs.airbyte.com/cloud/core-concepts#stream>stream</a>. Use this to define which files belong in the stream, their format, and how they should be parsed and validated. When sending data to warehouse destination such as Snowflake or BigQuery, each stream is a separate table.",
+        "order": 3,
+        "type": "array",
+        "items": {
+          "title": "SourceGCSStreamConfig",
+          "type": "object",
+          "properties": {
+            "name": {
+              "title": "Name",
+              "description": "The name of the stream.",
+              "order": 0,
+              "type": "string"
+            },
+            "globs": {
+              "title": "Globs",
+              "description": "The pattern used to specify which files should be selected from the file system. For more information on glob pattern matching look <a href=\"https://en.wikipedia.org/wiki/Glob_(programming)\">here</a>.",
+              "order": 1,
+              "type": "array",
+              "items": {
+                "type": "string"
+              }
+            },
+            "legacy_prefix": {
+              "title": "Legacy Prefix",
+              "description": "The path prefix configured in v3 versions of the S3 connector. This option is deprecated in favor of a single glob.",
+              "airbyte_hidden": true,
+              "type": "string"
+            },
```

Comment:
Looks like this can be removed since it's S3-specific.

Reply:
description updated